test/integration_test/snapshot_restore_test.go

```diff
@@ -0,0 +1,75 @@
+// +build integrationtest
+
+package integrationtest
+
+import (
+	"testing"
+	"time"
+
+	"github.com/portworx/torpedo/drivers/scheduler"
+	"github.com/stretchr/testify/require"
+)
+
+func testSnapshotRestore(t *testing.T) {
+	t.Run("simpleSnapshotRestoreTest", simpleSnapshotRestoreTest)
+	/*t.Run("groupSnapshotRestoreTest", groupSnapshotRestoreTest)
+	t.Run("cloudSnapshotRestoreTest", cloudSnapshotRestoreTest)
+	t.Run("groupCloudSnapshotRestoreTest", groupCloudSnapshotRestoreTest)*/
+}
+func createInPlaceRestore(t *testing.T, appKeys []string) []*scheduler.Context {
+	ctxs, err := schedulerDriver.Schedule("test",
+		scheduler.ScheduleOptions{AppKeys: appKeys})
+	require.NoError(t, err, "Error scheduling task")
+	require.Equal(t, 1, len(ctxs), "Only one task should have started")
+	return ctxs
+}
+
+func verifyInPlaceSnapshotRestore(t *testing.T, ctx []*scheduler.Context, startTime time.Time, timeout time.Duration) {
+	err := schedulerDriver.WaitForRunning(ctx[0], timeout, defaultWaitInterval)
+	// we shouldn't valid px specific things here just pass to torpedo validate inplacenspahostrestore
+	// in torpedo inplace snapshot restore for localsnapshot check alerts with volume id may be bit hard low prio
+	// in cloudsnaps check whether restore volume created and its ha level + alerts
+	require.NoError(t, err, "Error waiting for task")
+	err = schedulerDriver.ValidateVolumeSnapshotRestore(ctx[0], startTime)
+	require.NoError(t, err, "Error validating volumesnapshotRestore")
+}
+
+func simpleSnapshotRestoreTest(t *testing.T) {
+	// foldername + name as namespaces
+	snapCtx, err := schedulerDriver.Schedule("test",
```

Comment:
Use a specific name instead of "test" so that there are no conflicts in the future